content/03-reference/01-tools-and-interfaces/01-prisma-schema/index.mdx

```diff
@@ -159,6 +159,31 @@ dotenv.config() // load the environment variables
 console.log(`The connection URL is ${process.env.DATABASE_URL}`)
 ```
 
+### Use multiple .env files or .env files outside the prisma folder
+
+To use multiple `.env` files:
+
+1.  Install [dotenv-cli](https://www.npmjs.com/package/dotenv-cli):
+
+    ```copy
+    npm install -g dotenv-cli
+    ```
+
+2.  Create an `.env` file in, for example, the root of your project.
+
+3.  Include a reference to `dotenv` at the top of your project's entry file.
+
+    ```ts
+    import { config } from 'dotenv'
+    config()
+    ```
```

Comment:
Isn't this an alternative to the global installation of the dotenv CLI?

Reply:
@ryands17 Was there a specific reason why this was suggested to the person on Slack? Want to make sure I don't lose context.

Reply:
@mhwelander Yes it's an alternative if they do not want to use `dotenv-cli` in the command but directly in the file. I had given an example of using that while seeding the database initially.

Reply:
Ahh, then I misunderstood. To clarify:

1) Either you use dotenv -e .env3 in the command

OR

2) Include this in your file:

```
import { config } from "dotenv";
config({ path: ".env3" });
```

Is that right?

Reply:
Yep that is correct :)

Reply:
Fixed!